javascript/tables/src/__tests__/data/companies-headerless.csv

```diff
@@ -1,5 +1,6 @@
 1,Microsoft,160000,true
 2,Apple,150000,true
 3,Google,135000,true
+
```

Comment:
Is a blank line here intentional?

Reply:
yep, it tests that the parser ignores blank lines